src/Common/constants.tsx

```diff
@@ -622,32 +622,32 @@ export type CameraPTZ = {
 
 export const getCameraPTZ: (precision: number) => CameraPTZ[] = (precision) => [
   {
-    icon: "fa fa-arrow-up",
+    icon: "chevron-up",
     label: "Move Up",
     action: "up",
     loadingLabel: "Moving Up",
-    shortcutKey: ["Control", "Shift", "ArrowUp"],
+    shortcutKey: ["ArrowUp"],
   },
   {
-    icon: "fa fa-arrow-down",
+    icon: "chevron-down",
     label: "Move Down",
     action: "down",
     loadingLabel: "Moving Down",
-    shortcutKey: ["Control", "Shift", "ArrowDown"],
+    shortcutKey: ["ArrowDown"],
   },
   {
-    icon: "fa fa-arrow-left",
+    icon: "chevron-left",
     label: "Move Left",
     action: "left",
     loadingLabel: "Moving Left",
-    shortcutKey: ["Control", "Shift", "ArrowLeft"],
+    shortcutKey: ["ArrowLeft"],
   },
   {
-    icon: "fa fa-arrow-right",
+    icon: "chevron-right",
     label: "Move Right",
     action: "right",
     loadingLabel: "Moving Right",
-    shortcutKey: ["Control", "Shift", "ArrowRight"],
+    shortcutKey: ["ArrowRight"],
```

Comment:
umm... @skks1212 ... the updated shortcut icons are not visible...
Also I had 
![image](https://user-images.githubusercontent.com/25143503/174634326-dc6c5b5f-8e38-450e-82ec-5709df9e0a8a.png)

Reply:
The page is set to take up remaining space, so if there will be no space to scroll, the arrow key will not be an issue... I had changed it so that camera movement can be less complicated
I will add the question mark icon though

Reply:
@skks1212 try moving camera up from this page: https://deploy-preview-2758--care-net.netlify.app/assets/fd605386-90d5-4e8b-bb82-f62b7349d1d3/configure